File: seed/csharp-sdk/exhaustive/include-exception-handler/src/SeedApi.DynamicSnippets/Example34.cs
using SeedExhaustive;
using SeedExhaustive.Endpoints;

public partial class Examples
{
    public static async Task Example34()
    {
        var client = new SeedExhaustiveClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Params.GetWithInlinePathAndQueryAsync(
            new GetWithInlinePathAndQuery {
                Param = "param",
                Query = "query"
            }
        );
    }

}

File: seed/java-sdk/exhaustive/enable-public-constructors/src/main/java/com/snippets/Example34.java
package com.snippets;

import com.seed.exhaustive.SeedExhaustiveClient;
import com.seed.exhaustive.resources.endpoints.params.requests.GetWithInlinePathAndQuery;

public class Example34 {
    public static void main(String[] args) {
        SeedExhaustiveClient client = SeedExhaustiveClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpoints()
                .params()
                .getWithInlinePathAndQuery(GetWithInlinePathAndQuery.builder()
                        .param("param")
                        .query("query")
                        .build());
    }
}
